content/queues/javascript-apis.md

```diff
@@ -52,6 +52,11 @@ interface Queue<Body = any> {
   - Sends a message to the Queue. The body can be any type supported by the [structured clone algorithm](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Structured_clone_algorithm#supported_types), as long as its size is less than 128 KB.
   - When the promise resolves, the message is confirmed to be written to disk.
 
+- {{<code>}}sendBatch(body{{<param-type>}}any[]{{</param-type>}}){{</code>}} {{<type>}}Promise\<void>{{</type>}}
+
+  - Sends a batch of messages to the Queue. Each item in the array must be supported by the [structured clone algorithm](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Structured_clone_algorithm#supported_types). You can send up to 100 messages at a time, though items are limited to 128 KB each, and the total size of the array cannot exceed 256 KB.
```

Comment:
Kate will probably mention this once this PR moves to be against the main docs repo, but if you want to clean it up sooner then I think we generally avoid using the second person. The last sentence here could be rephrased to something like "A batch can contain up to 100 messages, though [...]`